pkg/cmd/roachprod/install/cockroach.go

```diff
@@ -436,10 +436,10 @@ func (h *crdbInstallHelper) generateStartCmd(
 		LogDir: h.c.Impl.LogDir(h.c, nodes[nodeIdx]),
 		KeyCmd: h.generateKeyCmd(nodeIdx, extraArgs),
 		Tag:    h.c.Tag,
-		EnvVars: append([]string{
+		EnvVars: append(append([]string{
 			"GOTRACEBACK=crash",
 			"COCKROACH_SKIP_ENABLING_DIAGNOSTIC_REPORTING=1",
-		}, h.getEnvVars()...),
+		}, h.c.Env...), h.getEnvVars()...),
```

Comment:
Just for reference, it looks like this used to happen in getEnvVars: https://github.com/cockroachdb/cockroach/pull/67363/files#diff-052e0342a18522387861c55bc7df04d8b6a888977f4593e2c8cd7c5749f94068L757-L761